### ISSUE TITLE
The re-implemented results are much lower than those in the paper

### ISSUE THREAD

**Issue by LoveMiki** [00why00/Glocal#13]
Hi,
I used the same dataset format of Mini-ImageNet and followed the instructions to run the code.
Specifically, I used the following command line from README:

python main.py --gpu 0 --max_epoch 100 --attention --energy --energy_loss --open_loss --lr 0.0001 --energy_method sum --pixel_wise --distance pixel_sim --pixel_conv --init_weights [/path/to/pretrained/weights] --dataset MiniImageNet --shot [number of shots] --ahead_combine --top_k [number of k]

I downloaded the pre-trained model and set -- shot 5 and --top_k 1.
However, I can only get the following test result:

$\mathbf{acc: 0.7198 + 0.0060}$ Energy: auroc=0.7458+0.0067, fpr95=0.7149+0.0133, auc-pr=0.7274+0.0070, f1 score=0.6869+0.0059 Logits: auroc=0.5009+0.0080, fpr95=0.9331+0.0055, auc-pr=0.5205+0.0058, f1 score=0.5003+0.0064 Combine: auroc=0.6786+0.0072, fpr95=0.7675+0.0118, auc-pr=0.6500+0.0069, f1 score=0.6348+0.0061 
best acc epoch 60, best auroc epoch 9, best val acc=0.7698 + 0.0061, best val auroc=0.7787 + 0.0060, total loss=1.8193, acc=0.7576 + 0.0060, energy auroc=0.7615+0.0070, energy fpr95=0.6496 + 0.0129, energy auc-pr=0.7372 + 0.0073, energy f1 score=0.6970 + 0.0060, logits auroc=0.4967+0.0094, logits fpr95=0.9202 + 0.0075, logits auc-pr=0.5192 + 0.0066, logits f1 score=0.4970 + 0.0070, $\mathbf{combine auroc=0.6843+0.0081}$, combine fpr95=0.7276 + 0.0120, combine auc-pr=0.6534 + 0.0076, combine f1 score=0.6342 + 0.0064

The accuracy of the closed-set and the open-set is 71.98% and 68.43%, respectively. These results are much lower than those in the paper (i.e., 83.05 for ACC and 82.29 for AUROC). 
Is it something going wrong in your code? By the way, could you provide the final model to validate the performance?

Thank you in advance.
Best.

**Comment by 00why00** [comment]
Sorry for the late response.

You may misunderstand the test result, and the real test result is ACC: 0.7198 AUROC: 0.7458.

By your next issue, you may have loaded the wrong pre-trained model, you can double check that the model is loaded correctly so that you can better reproduce our results.
